feat: add scripts to seed database (#1104)
aneoconsulting/ArmoniK: .docs/content/2.guide/populate-database.md

# Populate database

This guide will explain you how you can populate your ArmoniK database with **fake data**.

::alert{type="info"}
This guide is useful for the [Admin GUI](https://github.com/aneoconsulting/ArmoniK.Admin.GUI) or for testing purpose.
::

## Prerequisites

You need to have a running ArmoniK cluster. If you don't have one, please follow the [Getting Started](../1.installation/1.linux/1.installation.md) guide.

## Populate database

> In order to be easily populate database with [MongoDB scripts](https://www.mongodb.com/docs/mongodb-shell/write-scripts/) along the [@faker-js/faker](https://www.npmjs.com/package/@faker-js/faker) library.

Author: ```suggestion
In order to easily populate, we use [MongoDB scripts](https://www.mongodb.com/docs/mongodb-shell/write-scripts/) along the [@faker-js/faker](https://www.npmjs.com/package/@faker-js/faker) library.
```

## Going further

The section is only needed if you can't find your joy with current scripts.

We provide a strong base of utils scripts to write your own script.

First, you can check [current scripts](https://github.com/aneoconsulting/ArmoniK/tree/main/tools/mongodb) to see how it works.

Then, you can read the MongoDB documentation to learn how to write your own script.

Finally, we recommend you to use [execute-script.sh](https://github.com/aneoconsulting/ArmoniK/blob/main/tools/mongodb/execute-script.sh) to have the correct setup.

### Writing your own script

1. Create a file in the `mongodb` folder. Please name it with the following pattern: `<action>-<collection>.sh`.
2. Use the following command:

    ```sh
    # Get the directory of the script to create relative links
    DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
    # Description: <description with only a couple of words>
    "$DIR/utils/execute-script.sh" <script-name>
    ```

3. Create your JavaScript script in the `scripts` folder. Please name it with the following pattern: `<action>-<collection>.js`. (Same name as the file created in step 1)

> 4. Write your script. You can use the [@faker-js/faker](https://www.npmjs.com/package/@faker-js/faker) library to generate fake data.

Reviewer: Je pense que c'est un peu compliqué pour ce qu'on veut faire.
Néanmoins, je suis intéressé de voir ce que cela prodruit

Author: Tu préfères que j'écris un autre script (autre que les partitions pour voir ce que ça peut produire) dans cette PR ou qu'on merge et que je fasse une autre PR ?
